src/Microsoft.Extensions.FileSystemGlobbing/Util/StringComparisonHelper.cs

```diff
@@ -19,12 +19,10 @@ public static StringComparer GetStringComparer(StringComparison comparisonType)
                     return StringComparer.Ordinal;
                 case StringComparison.OrdinalIgnoreCase:
                     return StringComparer.OrdinalIgnoreCase;
-#if !NETSTANDARD1_3
```

Comment:
Can you replace this type with `StringComparer.FromComparison` (https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/StringComparer.cs#L76)?